app/init/emm_provider.ts

```diff
@@ -203,7 +203,7 @@ class EMMProvider {
                 translations[t('mobile.managed.blocked_by')].replace('{vendor}', this.vendor),
                 message,
                 buttons,
-                {cancelable: false, onDismiss: resolve},
+                {cancelable: false, onDismiss: () => resolve},
```

Comment:
Why this change?

Reply:
I set the 'strict' config to true in the 'tsconfig.json' and this was preventing me from committing.

onDismiss has a type of " (unknown) => void " and this is not assignable to type "() => void".

Reply:
It is related to what @migbot  commented in the first PR " Build failing due to TS error in app/init/emm_provider.ts "